### ISSUE TITLE
How to pre-train models using tiny-imagenet?

### ISSUE THREAD

**Issue by hobbitlzy** [NYU-DICE-Lab/selective_network_linearization#1]
I download the official tiny-imagenet and change the path of this [line](https://github.com/NYU-DICE-Lab/selective_network_linearization/blob/eac688cb8e50ebc0fe244e8478d3c174101cbeb5/datasets.py#L137) . But the accuracy  is always close to zero. Could you provide some guidelines to use the tiny-imagenet?

**Comment by ameya005** [comment]
Are you using [train_unstructured.py](https://github.com/NYU-DICE-Lab/selective_network_linearization/blob/main/train_unstructured.py) to pretrain the models? In that case, can you share the hyperaparams you are using so that I can help you debug this?

**Comment by hobbitlzy** [comment]
Yes, I keep all the hyperparams as default. The training works well on the dataset CIFAR100 but crashes on the Tiny-imagenet dataset.

**Comment by ameya005** [comment]
Can you share the logs that get printed when you run this? I have been able to train a tiny-imagenet model without an issue. I suggest first checking if the dataset has been downloaded correctly.

**Comment by hobbitlzy** [comment]
Thanks for your reply. The accuracy of the training set is normal, but it crashes on the test dataset. I have found the reason is the different format of the Tiny-ImageNet validation dataset.
